### DISCUSSION TITLE
Is the server GPU intensive ?

### DISCUSSION THREAD

**Discussion by arobase-che** [moyix/fauxpilot#87] [Q&A] [answered]
Hello,

After the training phase, does the server need the GPU ? If yes, does a client request is a GPU intensive task ?

Thank you !

**Comment by arobase-che** [comment, marked as answer]
Ok so it is from what I found.

Does co-pilote use a lot of GPU computational power too ?

**Reply by moyix** [reply]
Yep, it is generally very GPU intensive (both in VRAM required and how much work the GPU does). Copilot is also very GPU intensive, but because it's running the model on a bunch of servers at Microsoft it doesn't need to use your own GPU.

**Reply by arobase-che** [reply]
Thank you for you answer. :) 

I did expect the model generation to be GPU intensive but I'm a surprise by the fact that running the actual model is GPU intensive (in VRAM too !).

That made it a no-deal for my use case but thank you very much for making a free alternative to co-pilot.
